docs/sheets/GNSS.md

```diff
@@ -1,236 +1,251 @@
 # COMMON GNSS TASKS
 
-* :scroll: :one: [Creating or updating a GNSS __mark__](#mark)
-* :scroll: :two: [Creating or updating a GNSS __monument__](#monument)
-* :scroll: :three: [Installing or updating a GNSS __antenna__](#antenna)
-* :scroll: :four: [Installing or updating a GNSS __receiver__](#receiver)
-* :scroll: :five: [Installing or updating a GNSS __receiver__ firmware version](#firmware)
-* :scroll: :six: [Installing or updating a GNSS __receiver__ session configuration](#session) :dragon:
 * :hammer: [Combined editing steps](#steps)
+* :scroll: :one: [GNSS __receiver firmware version__ upgrade](#firmware)
+* :scroll: :two: [GNSS __receiver__ change](#receiver) :dragon: 
+* :scroll: :three: [GNSS __antenna__ change](#antenna) :dragon: 
+* :scroll: :four: [GNSS __receiver settings__ change](#session) :dragon:
+* :scroll: :five: [GNSS __coordinates__ update](#coordinates)
+* :scroll: :six: [__creating updating or closing__ a GNSS site](#openclose) :dragon:
+
 
 Global Navigation Satellite System (GNSS) equipment are installed at GNSS __marks__. Each __mark__ is
 attached to a __monument__. The details of the __mark__, as found in the __network/marks.csv__ file,
-tend to relate more to documentation, such as name and location. Whereas the __network/monuments.csv__ file
+tend to relate more to documentation, such as mark identification and location. Whereas the __network/monuments.csv__ file
 tends to list physical aspects of the actual monument as constructed.
 
 Equipment installed should be found in the the relevant __assets__ directory files. Actual times when
```

Comment:
"Equipment installed should be found in the the relevant" here "the" is repeated